src/pybind/mgr/dashboard/controllers/smb.py

```diff
@@ -349,6 +350,7 @@ def create(self, join_auth: JoinAuth) -> Simplified:
         """
         return mgr.remote('smb', 'apply_resources', json.dumps(join_auth)).to_simplified()
 
+    @raise_on_failure
     @CreatePermission
```

Comment:
woah, shouldn't this be `DeletePermission`?

Reply:
wow... I'll change it right away, thanks for pointing it out.